src/main/java/edu/usf/cutr/grha/ProcessorMain.java

```diff
@@ -42,7 +46,13 @@ public static void main(String[] args) {
         try {
             System.out.println("*** Chicago open TNC data ***");
             ChicagoTncParser chicagoTncParser = new ChicagoTncParser(new FileInputStream(args[1]));
-            IOUtils.printChicagoTncData(chicagoTncParser.parseFile());
+            // IOUtils.printChicagoTncData(chicagoTncParser.parseFile());
+            if (args.length == 3) {
+                new TncToGtfsWriter(args[2]).write(chicagoTncParser.parseFile());
+            } else {
+                new File(System.getProperty("user.dir") + "\\gtfs-dataset").mkdir();
```

Comment:
If you use the default `File()` constructor, I believe this should just create the file in the directory from which the JAR is being run, which would be my preference (rather than outputting to a different default directory).

Reply:
Sounds good. I can create a folder in the directory from where the jar file is run so that it looks more organized.

Reply:
"output" would be a good name.

Reply:
Also, be sure to use the Java `File.separator` instead of hard-coding the path separator so it works across platforms.

Reply:
So I tried with an empty constructor but I don't think it got exported to the `target` folder. Since I have the user directory I can route to the  `target` folder. Thoughts?

Reply:
Are you running from IntelliJ or from compiled JAR at command-line?

In either case I wouldn't expect it to get output to the `target` folder, unless `target` is your current folder.

You can see the gtfs-validator output here, which passes in a relative directory to the File constructor (`output`) and it will output to that subdirectory under your current directory you're running from - https://github.com/MobilityData/gtfs-validator/blob/master/main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java#L119. This same solution should work for us.

Reply:
Leaving the command-line option empty and outputting to `output` subdirectory works for me when running from IntelliJ "Run" button:

![image](https://user-images.githubusercontent.com/928045/111803080-16e4b200-88a5-11eb-860b-c0f9709d0d24.png)

So let's stick this with this solution. Maybe there is some kind of file or folder locking blocking execution on your machine?

One question, though - when providing the command-line parameter (i.e., top part of IF), shouldn't we also do `.mkdirs()` to ensure the directory exists before writing to it?

Reply:
Good catch. I always assumed the user will provide an existing directory to the tool.